README.md

```diff
@@ -62,6 +62,10 @@ Here is an example for the dialplan:
     exten => _X.,1,Dial(Dongle/r1/${EXTEN})
     exten => _X.,n,Hangup
 
+    exten => _X.,n,DongleSendSMS(dongle0,${EXTEN},"Please call me",1440,yes)
+
+    exten => _X.,n,DongleSendUSSD(dongle1,*#123#)
+
```

Comment:
All looks good to me, except for this bit.

The above is not sane dialplan, as these two are not reachable because of the hangup.

How about:

```
    [othersipserver-incoming]
    exten => _X.,1,Dial(Dongle/r1/${EXTEN})
    exten => _X.,n,Hangup()

    [send-sms]  
    exten => _X.,1,DongleSendSMS(dongle0,${EXTEN},"Please call me",1440,yes)

    [send-ussd]
    exten => _X.,1,DongleSendUSSD(dongle1,*#123#)
```